src/tests/utils/common.cairo

```diff
@@ -17,18 +18,27 @@ pub fn deploy_at(
     };
 }
 
+/// Deploys a contract from the class hash of another contract which is already deployed.
+pub fn deploy_another_at(
+    existing: ContractAddress, target_address: ContractAddress, calldata: Array<felt252>
+) {
+    let class_hash = get_class_hash(existing);
+    let contract_class = ContractClassTrait::new(class_hash);
+    deploy_at(contract_class, target_address, calldata)
+}
```

Comment:
This helper function will be helpful, it nearly solves the issue of not having `declare_if_not`